src/widgets/modalChildren/ADRInput.js

```diff
@@ -1,30 +1,90 @@
 /* eslint-disable react/forbid-prop-types */
-import React, { useState } from 'react';
-import { StyleSheet } from 'react-native';
+import React, { useEffect, useState, useLayoutEffect } from 'react';
+import { ActivityIndicator, StyleSheet, Text } from 'react-native';
 import { connect } from 'react-redux';
 import PropTypes from 'prop-types';
 import { JSONForm } from '../JSONForm/JSONForm';
 import { UIDatabase } from '../../database/index';
 import { FlexRow } from '../FlexRow';
+import { FlexView } from '../FlexView';
 import { PageButton } from '../PageButton';
-import { SUSSOL_ORANGE } from '../../globalStyles/colors';
+import { SUSSOL_ORANGE, WHITE } from '../../globalStyles/colors';
 import globalStyles from '../../globalStyles/index';
 import { PatientActions } from '../../actions/PatientActions';
 import { selectCurrentPatient } from '../../selectors/patient';
+import { useLocalAndRemotePatientHistory } from '../../hooks/useLocalAndRemoteHistory';
+import { dispensingStrings } from '../../localization';
 
-export const ADRInputComponent = ({ onCancel, onSave, patient }) => {
+const getSchemaItems = jsonSchema => {
+  const { properties = {} } = jsonSchema;
+  const { causes = {} } = properties;
+  const { items } = causes;
+  return items;
+};
+const mapHistory = history =>
+  history.map((h, index) => {
+    const { prescriberOrVaccinator, itemName, confirmDate } = h;
+    const vaccinator = prescriberOrVaccinator ? `  Vaccinator: ${prescriberOrVaccinator}` : '';
+
+    return `${index + 1}. ${itemName}  Date: ${confirmDate}${vaccinator}`;
```

Comment:
It seems like (unless I'm missing something) the timestamp isn't ever anything other than `T12:00:00.000Z` so it might look a bit neater with the time portion trimmed off before display?

Reply:
good point, thanks!
d308e80ba